components/Splitter.tsx

```diff
@@ -11,7 +14,7 @@ export function Splitter({
 }) {
   return (
     <div id="splitter" className="bright-hover tab" onClick={onClick}>
-      {isOpen ? '✗' : '\u{25c0}'}
+      {isOpen ? blackRightPointingTriangle : blackLeftPointingTriangle}
```

Comment:
I changed my mind on the x, I think a chevron or triangle works best here.

<table>
<tr>
	<th>Before
	<th>After
<tr>
	<td><img width="51" alt="Screenshot 13" src="https://github.com/user-attachments/assets/946f37c3-1f73-4670-b3be-f8e6b24dedfb"><td><img width="51" alt="Screenshot 14" src="https://github.com/user-attachments/assets/477341b8-b37e-4a30-9472-9baedd70ab04">
</table>

Reply:
No strong opinion on this.